daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/reader/InterfaceReader.scala

```diff
@@ -266,6 +288,9 @@ object InterfaceReader {
       ctx: Context): InterfaceReaderError \/ FieldWithType =
     type_(a.getType, ctx).flatMap(t => name(a.getField).map(_ -> t))
 
+  private def constructors(as: DamlLf1.DefDataType): InterfaceReaderError \/ ImmArray[Name] =
```

Comment:
Variants have constructors too.

```suggestion
  private def enumConstructors(as: DamlLf1.DefDataType): InterfaceReaderError \/ ImmArraySeq[Name] =
```

Reply:
done